metrics/prometheus.go

```diff
@@ -127,21 +131,19 @@ type Result struct {
 
 func parseKubectlOutput(output string) []Result {
 	lines := strings.Split(output, "\n")
-
 	var results []Result
 	for _, line := range lines {
-		o := strings.Split(line, " ")
-		if len(o) < 2 {
+		m := r.FindAllStringSubmatch(line, -1)
+		// Should be only 1 match, and should contain 4 elements (0: whole match, 1: resource-type, 2: name, 3: action
+		if len(m) != 1 || len(m[0]) != 4 {
+			log.Logger.Warn("Expected format: <resource-type>/<name> <action>", "line", line, "full output", output)
```

Comment:
The output returned by `kubectl` contains an empty line at the end, so this warning will be logged for every namespace on every run. 

You might want a
```go
if len(line) == 0 {
  continue
}
```
at the start of the loop. Or you could chomp the empty line out at the start of the function.

Reply:
Thinking about it, the tests should probably contain this line as well.